test/helper.js

```diff
@@ -24,5 +24,16 @@ function check (t, chunk, level, msg) {
   })
 }
 
+function check2 (t, chunk, wanted) {
```

Comment:
I did not dare replace the original `'check()'` with this new version, but that was the intent... Renamed it to `'checkGen'`